scripts/zkai_attention_kv_d8_fused_softmax_table_native_gate.py

```
@@ -0,0 +1,568 @@
+    return cases
+
+
+def run_gate() -> dict[str, Any]:
+    source_input = read_bounded_json(SOURCE_INPUT_JSON, MAX_SOURCE_INPUT_JSON_BYTES, "source input")
+    source_envelope = read_bounded_json(SOURCE_ENVELOPE_JSON, MAX_SOURCE_ENVELOPE_JSON_BYTES, "source envelope")
+    sidecar_envelope = read_bounded_json(SIDECAR_ENVELOPE_JSON, MAX_SIDECAR_ENVELOPE_JSON_BYTES, "sidecar envelope")
+    fused_raw = read_bounded_bytes(FUSED_ENVELOPE_JSON, MAX_FUSED_ENVELOPE_JSON_BYTES, "fused envelope")
+    if len(fused_raw) != FUSED_ENVELOPE_SIZE_BYTES:
+        raise AttentionKvD8FusedSoftmaxTableGateError(
+            f"fused envelope size drift: got {len(fused_raw)}, expected {FUSED_ENVELOPE_SIZE_BYTES}"
+        )
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)